doc/guides/kconfig/index.rst

```diff
@@ -1,918 +1,33 @@
 .. SPDX-License-Identifier: Apache-2.0
 
```

Comment:
Don’t need the SPDX license tag on doc files.

Reply:
Removed all the license tags. Thought it was required to please CI.